CustomShipSelect.cpp

```diff
@@ -3179,6 +3180,18 @@ HOOK_METHOD_PRIORITY(MenuScreen, OnRender, 1000, () -> void)
     }
 }
 
+HOOK_METHOD(MenuScreen, OnLanguageChange, () -> void)
+{
+    LOG_HOOK("HOOK_METHOD -> MenuScreen::OnLanguageChange -> Begin (CustomShipSelect.cpp)\n")
+    super();
+
+    if (G_->GetResources()->ImageExists("customizeUI/unlocks_disabled.png"))
+    {
+        unlocksDisabledTexture = G_->GetResources()->GetImageId("customizeUI/unlocks_disabled.png");
+        unlocksDisabledPrimitive = CSurface::GL_CreateImagePrimitive(unlocksDisabledTexture, 1106.f - unlocksDisabledTexture->width_ / 2, 104, unlocksDisabledTexture->width_, unlocksDisabledTexture->height_, 0.f, COLOR_WHITE);
```

Comment:
You're not destroying the old `unlocksDisabledTexture` and `unlocksDisabledPrimitive`, this will cause a memory leak.